index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="source/style.css" />
+        <meta name=viewport content="width=device-width, initial-scale=1">
+        </head>
+<body>
+    <div class="logo">My Logo</div>
+    <div class="box container"> 
+        <nav class="topnav">
+            <a href="#Portoflio">Portoflio</a>
+            <a href="#About">About</a>
+            <a href="#contact">Contact</a>
+         </nav>
+    </div>
+    <div class="box flex-container">
+        <div class="text">I’m Karam Glad to see you!</div>
+        <div class="text1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div><br>
+        <div class="text3">LET’S CONNECT</div>
+     </div>
```

Comment:
- [ ] The container of this section is a `<section>` tag. It is part of the project requirements and make your html more semantic  as explained above. Please kindly fix that.
- [ ] The main title (" I’m Karam Glad to see you! ") is a `<h1>` tag. Please kindly fix that.
- [ ] The paragraph (e.g., "I'm a software ...") is a `<p>` tag.
- [ ] The section should have fonts and colors (background and font) defined for the template 
- [ ] The content is aligned with flexbox
- [ ] The section has the margins (top, bottom, left, right) defined in the design, please compare the two designs below and kindly fix your implementation.
**Your Implementation**
![yoursat](https://user-images.githubusercontent.com/65852621/133893865-236a5ee2-fa5a-41e7-bd74-ba8a71d2d128.png)

**Your Implementation**
![sat1](https://user-images.githubusercontent.com/65852621/133893899-999d2a85-5e7f-4475-b2b8-55a86710aba4.png)